### ISSUE TITLE
Error : A conflict occurred for ALT+CTRL+H

### ISSUE THREAD

**Issue by yanntm** [eclipse/xtext-core#375]
Hi,

I've been getting this in traces and logs for some time.

```
!ENTRY org.eclipse.jface 2 0 2017-07-03 17:08:27.964
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2017-07-03 17:08:27.964
!MESSAGE A conflict occurred for ALT+CTRL+H:
Binding(ALT+CTRL+H,
	ParameterizedCommand(Command(org.eclipse.jdt.ui.edit.text.java.open.call.hierarchy,Open Call Hierarchy,
		Open a call hierarchy on the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@6f939588,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.ui.contexts.window,,,system)
Binding(ALT+CTRL+H,
	ParameterizedCommand(Command(org.eclipse.xtext.xbase.ui.hierarchy.OpenCallHierarchy,Open Call Hierarchy,
		Open a call hierarchy on the selected element,
		Category(org.eclipse.core.commands.categories.autogenerated,Uncategorized,Commands that were either auto-generated or have no category,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@2ba0948f,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.xtext.ui.XtextEditorScope,,,system)
Binding(ALT+CTRL+H,
	ParameterizedCommand(Command(org.eclipse.xtext.ui.editor.OpenCallHierarchy,Open Call Hierarchy,
		Open call hierarchy for the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@46b7c89b,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.xtext.ui.XtextEditorScope,,,system)
```

Seems to be the same issue reported here.
https://www.eclipse.org/forums/index.php/t/1078885/

Not sure if it actually causes problems in the editor apart from this pesky warning/error at launch.

**Comment by cdietrich** [comment]
Which xtext version do you use ?

**Comment by yanntm** [comment]
This is with 2.11 from neon update site. (v20170201-0242)

**Comment by cdietrich** [comment]
Looks like https://bugs.eclipse.org/bugs/show_bug.cgi?id=497070 which is fixed in 2.12

**Comment by yanntm** [comment]
great thanks, I'll make sure to update.
